sockeye/utils.py

```diff
@@ -32,7 +33,7 @@
 import numpy as np
 
 import sockeye.constants as C
-from sockeye import __version__
+from sockeye import __version__, constants as C
```

Comment:
Aren't you importing constants two times?

Reply:
yes, optimized imports